src/lib/client.ts

```diff
@@ -135,42 +136,43 @@ export const Client = async () => {
           client.do.blockSubscription.stop()
         },
         async start() {
-          if (!client.api) throw (Error("Api Missing, can't start block subscription yet."))
+          if (!client.api) throw (Error(`Api Missing, can't start block subscription yet.`))
```

Comment:
Why change to template string literal if there is no interpolation?

Reply:
Should i just use back slash on the **can't** single quote?

Reply:
Hm, is it required by linter? If not, I'd leave the way it was, but it is a minor nitpick.